Gaming The Syllabus/Assets/Scripts/Combat/UI/CombatUIManager.cs

```diff
@@ -41,6 +42,12 @@ public static void UpdateUnitHealthbar(CombatUnit unit, float currentHealth, flo
         combatUnitToHealthbar[unit].UpdateHealthbarValue(currentHealth, maximumHealth);
     }
 
+    public static void UpdateHealthbarText(CombatUnit unit, int amount) {
```

Comment:
Let's be a little more explicit about what amount is so idiots like me don't forget and screw it up